### ISSUE TITLE
 Why there are only 80 classes in `pred_logits`?

### ISSUE THREAD

**Issue by pdt0609** [lyuwenyu/RT-DETR#306]
**Star RTDETR**
请先在RTDETR主页点击**star**以支持本项目
Star RTDETR to help more people discover this project. 

---

Dear author @lyuwenyu,
I just have questions about the class classification of each queries: Why there are only 80 classes in `pred_logits`, while in DETR there remains 81 classes ( as background is counted as the 81st class).
Why your model can still work well with COCO while not considering the "background" class?
How RT-DETR can classify background class?

**Comment by lyuwenyu** [comment]
There is no `background class`. We use `vfl` as class loss.
